rich-default/environment.yml

```diff
@@ -0,0 +1,522 @@
+name: base
+channels:
+  - samoturk
+  - mordred-descriptor
+  - rdkit
+  - conda-forge
+  - defaults
```

Comment:
We really need to only install from conda-forge and defaults. Installing from these other channels is asking for trouble. Why do we need these others?

Reply:
It may also be good to set the channel priority to strict once we have only conda-forge and defaults.

Reply:
See: https://conda-forge.org/docs/user/tipsandtricks.html

Reply:
`mordred` is only available on `mordred-descriptor`
`rdkit` is available on conda-forge and I will look into using that instead.

Reply:
I think the order should be:

- conda-forge
- defaults
- other channels

This will ensure that we installe form conda forge first, then default, then the others for any missing packages. The channel strict priorty might work then too.

Reply:
Ha, I made the same comment below. At least set the channel order as I said.

I'd like to not have samoturk, mordred..., and rdkit though. I want us to stick to conda-forge and defaults to halep keep our sanity.